DS/MainThirdLargestItem.java

```diff
@@ -0,0 +1,57 @@
+//optional program
+import java.util.Scanner;
+class ThirdLargestItem {
+	public int[] addElement(int[] list,int listSize,Scanner sc) {
+		for(int i=0;i<listSize;i++)
+			list[i]=sc.nextInt();
+		return list;
+	}
+	public int getThirdLargest(int[] list) {
+		for(int i=0;i<list.length;i++) {
+		for(int j=0;j<list.length-1-i;j++) {
+			if(list[j]>list[j+1]) {
+				int swapVariable;
+				swapVariable=list[j];
+				list[j]=list[j+1];
+				list[j+1]=swapVariable;
+				}
+			}
+		}
+		int counter=0,thirdLargest=list[0];
+		for(int i=0;i<list.length-1;i++) {
```

Comment:
When your list is already sorted, what this loop will do? Can't we just return list[2] in that case?
Any way, try to write a method which will only have one loop and find third largest number, no extra loop, no nested loop.